front/app/modules/commercial/representativeness/admin/components/ChartCard/utils.ts

```diff
@@ -1,9 +1,5 @@
-import { format } from 'd3-format';
-
 export const formatPercentage = (percentage: number) => `${percentage}%`;
 
-export const formatThousands = format(',');
```

Comment:
Discovered that you can do this with Native JS (`Number.prototype.toLocaleString()`) so removed the d3 dependency

Reply:
should't we use the user locale instead?

Reply:
That would be perfect, but the problem is in `modules/commercial/representativeness/admin/components/Field/Options/utils.ts`. Converting the number to a locale string is one thing, but there we have to convert the locale string back to a number. Unfortunately there is no easy way to do that, so for now I am just using the American formatting for everything